Gemfile

```diff
@@ -73,6 +73,7 @@ gem "synaccess_connect", "0.2.2", github: "tablexi/synaccess"
 
 group :development do
   gem "coffeelint"
+  gem "haml_lint", github: "brigade/haml-lint", ref: "1384ec"
```

Comment:
The `0.23.2` version had a bug that prevented us from excluding items in the `.haml-lint_todo.yml` from the build. The issue was fixed on master, but a new version hadn't yet been released.

Looks like they just released `0.24.0` this morning with the fix, so I am updating to use that release.